package.json

```diff
@@ -34,5 +34,7 @@
             "pre-commit": "npm uninstall husky"
         }
     },
-    "dependencies": {}
+    "dependencies": {
```

Comment:
Why not devDependencies?

Reply:
because DefinitelyTyped isn't on npm so nothing can take a dependency on it? But I guess it makes more sense there.